srv/modules/runners/validate.py

```diff
@@ -413,9 +413,12 @@ def _monitor_check(self, name):
             self.errors.setdefault(name, []).append(msg)
         elif same_hosts:
             count = len(same_hosts.keys()[0].split(","))
-            if count < 3:
-                msg = "Must have at least three entries"
-                self.errors[name] = [ msg ]
+            if (not self.in_dev_env and count < 3) or (self.in_dev_env and count < 1):
+                if self.in_dev_env:
+                    msg = "Must have at least one monitor"
+                else:
+                    msg = "Must have at least three monitors"
+            self.errors[name] = [ msg ]
```

Comment:
I think this statement must be inside the first conditional block, otherwise you'll be adding an error message even if everything is ok